### PR TITLE
fix password validation bug -now validation works fine

### PR DESCRIPTION
**Pull request by greentoss**
bug3

fix password validation bug -now validation works fine
also button submit disabled untill all inputs in form are filled and verifyed

### CODE REVIEW

**Review by Radomyr-kh — APPROVED** [review]

File: src/utils/validations/login.js [hunk 1]
```diff
@@ -30,8 +30,8 @@ export const lastName = (value) => {
 }
 
 export const confirmPassword = (password, data) => {
-  let helperText
-  if (password !== data.confirmPassword) {
+  let helperText = ''
+  if (password !== data.password) {
     helperText = 'common.errorMessages.passwordsDontMatch'
   }
   helperText = emptyField(password, helperText)
```
Comment (MickhaelSh) [inline src/utils/validations/login.js:37]: remove this assigning , just return emptyField(password, helperText)

**Review by Tolik170 — APPROVED** [review]

File: src/utils/validations/login.js (hunk 1 above)
Comment (greentoss) [reply src/utils/validations/login.js]: fixed

**Review by MickhaelSh — APPROVED** [review]

File: src/utils/validations/common.js [hunk 2]
```diff
@@ -25,3 +17,47 @@ export const textField = (min, max) => (value) => {
...
+      }
+    }
+  }
+
+  for (const validationsKey in validations) {
+    if (marker === validationsKey) {
+      validations[validationsKey](value)
+    }
+  }
+
```
Comment (maarekua) [inline src/utils/validations/common.js:58]: return emptyField(password, validations[marker](value))

Comment (greentoss) [reply src/utils/validations/common.js:47]: fixed

**Review by maarekua — APPROVED** [review]

**Pull request merged**